### DISCUSSION TITLE
Microcode script question

### DISCUSSION THREAD

**Discussion by CoreyJ87** [tteck/Proxmox#1889] [Q&A] [answered]
Hey guys I have an intel i7 5930k. I ran the microcode update script and I got these 3 options
![image](https://github.com/tteck/Proxmox/assets/247750/a4efd775-c8ff-40d8-9547-2f2f73731bce)
Whats the difference in the deb12UL one vs just the regular amd64 one? Also, should I be worried that it does not currently find a microcode revision? Is that just because I've never installed one?

**Comment by tteck** [comment, marked as answer]
In the realm of package version comparison, a package that includes "~deb12UL" in its version string (such as `intel-microcode_3.20230808.1~deb12UL_amd64.deb`) would typically be regarded as having a lower precedence or a lower version number compared to a package that lacks the "~deb12UL" portion. This is how I interpret it, at least.

**Reply by CoreyJ87** [reply]
@tteck okay awesome. Thanks for the quick reply. So I should probably just use the second one in that list then correct? Also, I assume its okay that its not detecting a current microcode revision? Is that just because I've never installed any?

Also side note: Thanks for all your hard work on these scripts. They are IMMENSELY helpful. I've been telling everyone about them lol.

**Reply by tteck** [reply]
Yes, Yes, Yes and thank you. 😉

**Reply by CoreyJ87** [reply]
Awesome thanks again.
